packages/mui-base/src/ModalUnstyled/ModalManager.ts

```diff
@@ -31,6 +31,31 @@ function getPaddingRight(element: Element): number {
   return parseInt(ownerWindow(element).getComputedStyle(element).paddingRight, 10) || 0;
 }
 
+function isAriaHiddenForbiddenOnSibling(sibling: Element): boolean {
```

Comment:
```suggestion
function isAriaHiddenForbiddenOnElement(element: Element): boolean {
```

Reply:
This name for the method makes more sense. The fact that it is used on a sibling element is just in the context where it is invoked.